src/System.Diagnostics.PerformanceCounter/src/System.Diagnostics.PerformanceCounter.csproj

```diff
@@ -47,11 +47,11 @@
     <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Kernel32\Interop.FormatMessage.cs">
       <Link>Common\CoreLib\Interop\Windows\Interop.FormatMessage.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\advapi32\Interop.RegCloseKey.cs">
+    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Advapi32\Interop.RegCloseKey.cs">
```

Comment:
Same here.

Reply:
which do we prefer?

Reply:
I would prefer lowercase, and that seem to be the majority of the cases, but I don't know what other people think on this.

cc @jkotas @stephentoub @danmosemsft

Reply:
The casing of the filename should match the casing of the `class` that the file or directory contains.

The interop classes start with upper case: `class Kernel32`, `class Advapi32`, ... .

Reply:
The upper case is much more common, e.g.:

`Kernel32` 1571 hits over 415 files
`kernel32` 616 hits over 96 files

Reply:
We should use the upper case.